solutions/swb-ui/infrastructure/src/redirectFunction.js

```diff
@@ -1,25 +0,0 @@
-/***************************************************************************************************************************************************************************************
```

Comment:
I think there is a ticket related to change this file to .ts, I will remove it from the backlog